data/hai/unfettered 0 prologue.txt

```diff
@@ -297,6 +297,107 @@ mission "Unfettered Jump Drive 3"
+			label weapons
+			choice
+				`	"Tell me more about that Tripulse Cannon I can buy."`
+					goto tripulse
+				`	"Can you tell me more about that ionic small weapon?"`
+				`	"I'd like to learn about the Ion Turret."`
+					goto turret
+				`	"I would just prefer the money, actually."`
+					goto money
+			label blaster
+			`	"Ah, the Blaster. It is a favored by many Shield Beetle captains, because it can easily help them turn the tide of battle, without needing to dedicate too much weapon space for it. It is true that it does not do enough ion damage to properly disable the enemy ship, but it is able to jam their weapons and give you the upper hand in a long fight, especially since its damage potential is acceptable."`
+			scene "outfit/hai ionic blaster"
+				goto backtochoose
+			label turret
+			`	"That is my preferred weapon out of all of them, by far. Unlike the ion cannon that is a hybrid weapon meant to be able to damage the enemy from long range whilst applying a good deal of ion, this one's role is mostly to disable enemy ships. Even a ship with high batteries will cower before this weapon! And due to it being a high range turret, fighters and small ships will never bother you again!"`
+			scene "outfit/hai ionic turret"
+			label backtochoose
+			`	"However, we only have these weapons in very limited quantity, and as such we cannot afford to give these to you yet. So, about my offer, have you made up your mind?"`
+				goto weapons
+
+			label money
+			`	"Oh. Well, in that case, we will not bother you again with this weapon, or any other, Captain, here or in the job board."`
+			choice
+				`	"Sounds good to me."`
+				`	"Wait, what was that weapon again?"`
+					goto weapons
+			action
+				payment 1000000
+				set "unfettered: money for drives"
+			`	"As you wish. Do remember that if you want to sell further jump drives to us for money you may go to the job board."`
+				accept
+
+			label tripulse
+			`	"The tripulse shredder may seem crude, but it packs a punch and its specialty is how it combines a lot of firepower in one gun slot, and even allows for hit and run tactics due to its burst nature. Some captains even say that a Sea Scorpion can take out a Shield Beetle if it uses 4 of them, if you know what you're doing, of course! As I said, I may offer you two of these in exchange for your jump drive."`
+			scene "outfit/tripulse shredder"
+			choice
+				`	"I'll take that offer."`
+				`	"I'd like to learn more about the other weapons first."`
+					goto weapons
+				`	"Actually, could you give me the money instead?"`
+					goto money
+			action
+				outfit "Tripulse Shredder" 2
+			`	They order their crew to do the transfer, and come address you before leaving. "Do note that if you ever want more weapons we will always be willing to trade them for jump drives in the job board. We can of course always offer money instead if that is of higher value to you. Of course, we can only do this exchange here, on <origin>."`
```

Comment:
```suggestion
			`	They order their crew to do the transfer, and come address you before leaving. "Do note that if you ever want more weapons, we will always be willing to trade them for jump drives in the job board. We can of course always offer money instead if that is of higher value to you. Of course, we can only do this exchange here, on <origin>."`
```
Also not sure about the repeated 'of course'.

Reply:
Fixed it